File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/CurrentPlayerIDs.scala

package org.openapitools.client.model


case class CurrentPlayerIDs (
    /* The player UUID without hyphens */
    _id: String,
    /* The current name being used by this player */
    _name: String,
    /* If account has not been converted */
    _legacy: Option[Boolean],
    /* If the player has not puchased the game */
    _demo: Option[Boolean]
)
object CurrentPlayerIDs {
    def toStringBody(var_id: Object, var_name: Object, var_legacy: Object, var_demo: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name,"legacy":$var_legacy,"demo":$var_demo
        | }
        """.stripMargin
}

File: generated-sources/php-slim/mojang-api/lib/com/github/asyncmc/mojang/api/php/slim/model/CurrentPlayerIDs.php
<?php
/**
 * CurrentPlayerIDs
 */
namespace com.github.asyncmc.mojang.api.php.slim.server\com.github.asyncmc.mojang.api.php.slim.model;

/**
 * CurrentPlayerIDs
 */
class CurrentPlayerIDs
{
    
    /** @var string $id The player UUID without hyphens*/
    private $id;
    
    /** @var string $name The current name being used by this player*/
    private $name;
    
    /** @var bool $legacy If account has not been converted*/
    private $legacy;
    
    /** @var bool $demo If the player has not puchased the game*/
    private $demo;
}
